shared-bindings/busio/SPI.c

```diff
@@ -233,7 +233,7 @@ STATIC mp_obj_t busio_spi_write(size_t n_args, const mp_obj_t *pos_args, mp_map_
     normalize_buffer_bounds(&start, args[ARG_end].u_int, &length);
 
     if (length == 0) {
-        mp_raise_ValueError("Buffer must be at least length 1");
```

Comment:
Correct the doc at line 210. Maybe say: "If the buffer or slice is of zero length, nothing will happen." or some better phrasing.